shared/haml/course_explorer_table.haml

```diff
@@ -216,9 +245,10 @@
               .arrow
                 %i.arrow-down{class: "fa fa-caret-down"}
                 %i.arrow-up{class: "fa fa-caret-up"}
-              - if course[:range]
+              - if course[:range] || course[:announcement]
```

Comment:
Should this be `course[:range] && course[:announcement]`? Are both being displayed here?

Reply:
since the previous logic was `if course[:range]` i wanted to allow for the possibility that there is no course range (even though all our courses have ranges). we need the `%br/` if either value is present, but we display them next to each other and if one is absent it renders nothing. so this should read: "if we have a range or an announcement, add a <br>, then try to render both".

Reply:
i realize this code didn't make much sense without a screenshot of the responsive view! which i just added to the description.